tests/test_forms.py

```diff
@@ -278,3 +278,17 @@ class MyModel(Base):
 
     assert isinstance(Form()._fields["email"], EmailField)
     assert isinstance(Form()._fields["number"], IntegerField)
+
+
+@pytest.mark.skipif(engine.name != "postgresql", reason="PostgreSQL only")
+async def test_form_with_uuids() -> None:
+    class UUIDModel(Base):
+        __tablename__ = "uuid_model"
+
+        id = Column(UUID(as_uuid=True), primary_key=True)
+        team_id = Column(UUID(as_uuid=True))
+
+    Form = await get_model_form(model=UUIDModel, engine=engine)
+
+    assert len(Form()._fields) == 1
+    assert isinstance(Form()._fields["team_id"], NullableStringField)
```

Comment:
Can we add the asserts to the existing test for postgres: test_model_form_postgresql ?

Reply:
done